app-creator/src/widgets/actions-panel/actions-panel.ts

```diff
@@ -82,21 +92,24 @@ export class ActionsPanel extends LitElement {
     const widgetsMarkup =
       selectedTab === 1 ? html`<widgets-tab></widgets-tab>` : nothing;
 
+    const attributesMarkup =
+      selectedTab === 2 ? html`<attributes-tab></attributes-tab>` : nothing;
```

Comment:
This is better expressed as a switch statement:

```
let renderedTab = nothing;
switch (selectedTab)
  case Tab.widgets:
   ...
  case Tab.templates:
   ...
}
```

Then just render `renderedTab` below.